File: aoj/ITP1/02/2_D.cpp
#include<iostream>

using namespace std;

int main(){
    int w,h;
    int x,y;
    int r;

    cin >> w >> h >> x >> y >> r;

    if(x-r < 0 
    || w < x + r
    || y-r < 0
    || h < y + r)
        cout << "No" << endl;

    else 
        cout << "Yes" << endl;

    return 0;
}

File: aoj/golang/ITP1/02/2_D.go
package main

import (
    "fmt"
)

var (
    w, h int
    x, y int
    r    int
)

func main() {
    fmt.Scanf("%d %d %d %d %d", &w, &h, &x, &y, &r)

    if (x - r < 0) ||
       (w < x + r) ||
       (y - r < 0) ||
       (h < y + r) {
        fmt.Println("No")
    } else {
        fmt.Println("Yes")
    }
}
